namespaces/ci_job.yml

```diff
@@ -16,12 +16,12 @@ namespace:
     description: >
       Name of the CI job.
 
-  - name: build_id
+  - name: number
```

Comment:
`build_id` seems a lot more descriptive than `number` - is this necessary to change?